safe/test/utilities.py

```diff
@@ -426,6 +426,7 @@ def load_test_raster_layer(*args, **kwargs):
     .. versionadded:: 4.0
     """
     path = standard_data_path(*args)
+    assert os.path.exists(path), 'Path %s is not exist' % path
```

Comment:
Ah yes, you are right about the English. Sometimes my English is off when there is '%s'. :D Thanks.

Well, we can. But, the main point of this to see in travis which path is not exist.